packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.slow-spec.ts

```diff
@@ -255,6 +255,19 @@ function compareSearchResults(expected: SearchInWorkspaceExpectation[], actual:
 describe('ripgrep-search-in-workspace-server', function (): void {
     this.timeout(10000);
 
+    // Ignoring search with query provided.
+    it('ignore files with ignore query as "potatoes', done => {
```

Comment:
The naming of the test-case does not follow the pattern:

```ts
it('should....`
```

The name also does not reflect the test-case well (we are not searching for `potatoes` and there is no mention of ignoring files which is what the test-case is attempting to cover). We read these entries in the test report and the following name is not accurate.